README.md

```diff
@@ -63,11 +63,11 @@ To build and test in Visual Studio, open
 Alternatively, you can build REEF.NET from a developer command line
 via:
 
-    msbuild .\lang\cs\Org.Apache.REEF.sln
+    msbuild .\lang\cs\Org.Apache.REEF.sln /p:Configuration="Debug" /p:Platform="x64"
 
 To test, execute the following command thereafter:
 
-    vstest.console.exe .\lang\cs\bin\x64\Debug\Org.Apache.REEF.Tests\Org.Apache.REEF.Tests.dll /Platform:x64
+    msbuild .\lang\cs\TestRunner.proj /p:Configuration="Debug" /p:Platform="x64"
```

Comment:
So, tests are a new project now? I always thought that we could have something like `msbuild /t:test`. Is that not possible?